src/iop/lens.cc

```diff
@@ -342,7 +347,23 @@ static lfModifier * get_modifier(int *mods_done, int w, int h, const dt_iop_lens
   int mods_todo = d->modify_flags & mods_filter;
   int mods_done_tmp = 0;
 
-#ifdef LF_0395
+#ifdef LF_2000
+  mod = new lfModifier(d->lens, d->focal, d->crop, w, h, LF_PF_F32) ; //Reverse = false
+  if(mods_todo & LF_MODIFY_DISTORTION)
+	  mods_done_tmp |= mod->EnableDistortionCorrection();
+  if((mods_todo & LF_MODIFY_GEOMETRY) && (d->lens->Type != d->target_geom))
+	  mods_done_tmp |= mod->EnableProjectionTransform(d->lens->Type);
+  if((mods_todo & LF_MODIFY_SCALE) && (d->scale != 1.0))
+	  mods_done_tmp |= mod->EnableScaling(d->scale);
+  if(mods_todo & LF_MODIFY_TCA)
+  {
+    if(d->tca_override) mods_done_tmp |= mod->EnableTCACorrection(d->custom_tca);
+    else mods_done_tmp |= mod->EnableTCACorrection();
+  }
+  if(mods_todo & LF_MODIFY_VIGNETTING)
+	  mods_done_tmp |= mod->EnableVignettingCorrection(d->aperture, d->distance);
+
+#elif LF_0395
```

Comment:
Why not use `#if defined(LF_0395) || defined(LF_2000)` instead of duplicating the entire block of code?

Reply:
I'll fix that.